packages/utils/src/timeoutPromise.ts

```diff
@@ -0,0 +1,2 @@
+export const timeoutPromise = (timeout: number) =>
```

Comment:
I would suggest slightly more explicit name `createTimeoutPromise`

Reply:
Changed to the suggested d1aff1ecff46ed7647566eb73060a75b50ec2c00